modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -58,6 +58,15 @@ export default function AuditBarChart({rtl, vocabularies}) {
 			return acc.concat(newBar);
 		}, []);
 
+		// Reorder bars to push 'none' category to the end
```

Comment:
(Not being sarcastic here) but I don't think I would have known what this does without a comment. 😂

Reply:
Are comments like this allowed in DXP now? There was a time when we were told to remove all comments since the "code should explain itself".

_note: I am 100% pro-comments_

Reply:
I agree with you @bryceosterhaus, I prefer an understandable code than a thousand of comments. I'll try to simplify it or write it in another way

Reply:
Originally commented [here](https://github.com/liferay-frontend/liferay-portal/pull/733/files/4d9ed1ef47c334fd3607bf321b13c3082cec6503..c7d151884ceb707256ea35d1366dde60716f6a42#r565907367), so "moving" it here.

---

For me at least, just splitting it into separate steps would have made it easier to grok:

```js
const noneIndex = bars.findIndex((bar) => bar.dataKey === 'none');

const noneItem = bars.splice(noneIndex, 1);

bars = bars.concat(noneItem);
```

Reply:
That's what I was doing right now, split the logic 😄  Thanks!